backend/src/controllers/v2/secretsController.ts

```diff
@@ -830,204 +859,211 @@ export const updateSecrets = async (req: Request, res: Response) => {
         }
     }
     */
-    const channel = req.headers?.['user-agent']?.toLowerCase().includes('mozilla') ? 'web' : 'cli';
-
-    interface PatchSecret {
-        id: string;
-        secretKeyCiphertext: string;
-        secretKeyIV: string;
-        secretKeyTag: string;
-        secretValueCiphertext: string;
-        secretValueIV: string;
-        secretValueTag: string;
-        secretCommentCiphertext: string;
-        secretCommentIV: string;
-        secretCommentTag: string;
-        tags: string[]
-    }
-
-    const updateOperationsToPerform = req.body.secrets.map((secret: PatchSecret) => {
-        const {
+  const channel = req.headers?.["user-agent"]?.toLowerCase().includes("mozilla")
+    ? "web"
+    : "cli";
+
+  interface PatchSecret {
+    id: string;
+    secretKeyCiphertext: string;
+    secretKeyIV: string;
+    secretKeyTag: string;
+    secretValueCiphertext: string;
+    secretValueIV: string;
+    secretValueTag: string;
+    secretCommentCiphertext: string;
+    secretCommentIV: string;
+    secretCommentTag: string;
+    tags: string[];
+  }
+
+  const updateOperationsToPerform = req.body.secrets.map(
```

Comment:
Looks like we are not allowing user to update the folder id of a secret via api for now? I think should be fine for the time being